v20.2/export.md

```diff
@@ -30,14 +30,18 @@ Only members of the `admin` role can run `EXPORT`. By default, the `root` user b
 
  Parameter | Description
 -----------|-------------
- `file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported CSV data.<br><br>Note: Exports do not generate unique names across exports, so each export should have a unique destination to avoid overwriting.
+ `file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported CSV data.<br><br>Note: It is best practice to use a unique destination for each export, to avoid the potential for multiple exports in one directory.
```

Comment:
nit: ...to avoid mixing files from different exports in one directory. (or something along these lines)

Should we also add that the files returned on a successful EXPORT could be used to identify what was written?

Reply:
I’ve documented the return values [here](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/2eb81a63180981953057abcaf9ae195d5f32b3b5/stable/export.html#success-responses). Look good?

I tweaked the phrasing as you mention as well.

Reply:
LGTM, thanks!